packages/scene-composer/src/components/panels/scene-settings/SceneBackgroundSettingsEditor.tsx

```diff
@@ -0,0 +1,85 @@
+import useLifecycleLogging from '../../../logger/react-logger/hooks/useLifecycleLogging';
+import { useStore } from '../../../store';
+import { ColorSelectorCombo } from '../scene-components/tag-style/ColorSelectorCombo/ColorSelectorCombo';
+
+export const SceneBackgroundSettingsEditor: React.FC = () => {
+  useLifecycleLogging('SceneBackgroundSettingsEditor');
+
+  const sceneComposerId = useContext(sceneComposerIdContext);
+  const intl = useIntl();
+  const setSceneProperty = useStore(sceneComposerId)(
+    (state) => state.setSceneProperty<ISceneBackgroundSetting | undefined>,
+  );
+  const backgroundSettings = useStore(sceneComposerId)((state) =>
+    state.getSceneProperty<ISceneBackgroundSetting>(KnownSceneProperty.SceneBackgroundSettings),
+  );
+
+  const [backgroundEnabled, setBackgroundEnabled] = useState(!!backgroundSettings);
+  const [internalColor, setInternalColor] = useState(backgroundSettings?.colorHex || 0xcccccc);
+  //todo work with UX to determine if fog and background colors should share color pool with tags
+  const tagStyleColors = useStore(sceneComposerId)((state) =>
+    state.getSceneProperty<string[]>(KnownSceneProperty.TagCustomColors, []),
+  );
+  const setTagColorsSceneProperty = useStore(sceneComposerId)((state) => state.setSceneProperty<string[]>);
+
+  const onToggleBackground = useCallback(
+    (checked: boolean) => {
+      if (checked) {
+        setSceneProperty(KnownSceneProperty.SceneBackgroundSettings, {
+          colorHex: internalColor,
+        });
+      } else {
+        setSceneProperty(KnownSceneProperty.SceneBackgroundSettings, undefined);
+      }
+      setBackgroundEnabled(checked);
+    },
+    [backgroundSettings, internalColor],
+  );
+
+  const onColorChange = useCallback(
+    (color: string) => {
+      const hex = parseInt(color.replace(/^#/, ''), 16);
+      if (hex !== internalColor) {
+        setInternalColor(hex);
+        if (backgroundEnabled) {
+          setSceneProperty(KnownSceneProperty.SceneBackgroundSettings, {
+            colorHex: hex,
+          });
+        }
+      }
+    },
+    [backgroundEnabled, internalColor],
+  );
+
+  return (
+    <React.Fragment>
+      <SpaceBetween size='s'>
+        <Checkbox
+          data-testid='enable-background-checkbox'
+          checked={!!backgroundSettings}
+          onChange={(e) => onToggleBackground(e.detail.checked)}
+        >
+          {intl.formatMessage({ defaultMessage: 'Enable Background', description: 'checkbox label' })}
+        </Checkbox>
+        {!!backgroundSettings && (
+          <ColorSelectorCombo
+            color={`#${internalColor.toString(16).padStart(6, '0')}`}
+            onSelectColor={(pickedColor) => onColorChange(pickedColor)}
+            onUpdateCustomColors={(chosenCustomColors) =>
+              setTagColorsSceneProperty(KnownSceneProperty.TagCustomColors, chosenCustomColors)
+            }
+            customColors={tagStyleColors}
```

Comment:
same here for whether scene background color should share the same custom colors list with tag

Reply:
see above, added backgroundCustom save separate from tagCustom